doc/source/notebooks/advanced/convolutional.ipynb

```diff
@@ -367,19 +325,22 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Train acc: 100.0%\n",
-      "Test acc : 98.33333333333333%\n",
-      "conv elbo after training: -4.4511e+01\n"
+      "Train acc: 97.0%\n",
+      "Test acc : 98.0%\n",
+      "conv elbo after training: -4.2114e+01\n"
```

Comment:
why is this different now ?

Reply:
Different seed? It is a common problem, that we often use np.random without caring about seed and etc., but fixing it is out of the scope of this PR